.github/actions/helm-default/action.yaml

```diff
@@ -25,10 +29,8 @@ runs:
         echo sha=${{ inputs.image-tag }} >> $GITHUB_OUTPUT
 
         CILIUM_INSTALL_DEFAULTS="--chart-directory=${{ inputs.chart-dir }} \
-          --helm-set=debug.enabled=true \
-          --helm-set=debug.verbose=envoy \
-          --helm-set=debug.metricsSamplingInterval=30s \
           --helm-set=hubble.relay.retryTimeout=5s \
+          --helm-set=debug.metricsSamplingInterval=30s \
```

Comment:
Nit. I'd probably move it to the debug section as well, given the flag name.

Reply:
Actually had it there initially but moved it out as it's low overhead and useful in benchmark runs too.